presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution.property;
+
+import com.facebook.airlift.configuration.Config;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+public class NativeExecutionSystemConfig
+{
+    private static final String CONCURRENT_LIFESPANS_PER_TASK = "concurrent-lifespans-per-task";
```

Comment:
Is this also C++ config?

Reply:
This is used in Presto Cpp yes. But this is a common property also used in Java

Reply:
same, let's add comment to indicate the place in C++ where these configs are used